data/configs/default.json

```diff
@@ -7,14 +7,40 @@
       "type": "Other",
       "icon": "https://c.tenor.com/o656qFKDzeUAAAAC/rick-astley-never-gonna-give-you-up.gif",
       "url": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
+    },
+    {
+      "id": "b8221afa-6064-43a5-8eed-c78e7af94e27",
+      "type": "NZBGet",
+      "name": "NZBGet",
+      "icon": "https://cdn.jsdelivr.net/gh/walkxcode/dashboard-icons/png/nzbget.png",
+      "url": "http://192.168.1.61:6789",
+      "username": "jonjon1123",
+      "password": "aRsBPQfVxN3u@@5",
```

Comment:
Ouch...

Reply:
![](https://media.tenor.com/vCOInrRvG5UAAAAS/face-palm-oh-no.gif)

Reply:
Yep :( Def accidentally committed the config file... Password has been changes so all is good.

Reply:
I don't know why you resolved this. The secret is still here and must be removed!